ufm_devices/reboot_action.py

```diff
@@ -0,0 +1,86 @@
+import os
+import logging
+from ufm_devices.ufm_devices_action import UfmDevicesAction, ActionConstants
+
+try:
+    from utils.utils import Utils
+    from utils.args_parser import ArgsParser
+    from utils.config_parser import ConfigParser
+    from utils.logger import Logger, LOG_LEVELS
+except ModuleNotFoundError as e:
+    print("Error occurred while importing python modules, "
+          "Please make sure that you exported your repository to PYTHONPATH by running: "
+          f'export PYTHONPATH="${{PYTHONPATH}}:{os.path.dirname(os.getcwd())}"')
+
+
+
+class RebootActionConstants:
+    ACTION = "reboot"
+    args_list = [
+        {
+            "name": f'--{ActionConstants.API_OBJECT_IDS}',
+            "help": "comma separated devices GUIDs if",
+        },
+        {
+            "name": f'--{ActionConstants.UFM_API_DESCRIPTION}',
+            "help": "action description",
+        }
+    ]
+
+
```

Comment:
username and password should be provided as arguments, 
also help message should be reviewed